x-pack/plugins/security_solution/public/overview/components/overview_cti_links/cti_disabled_module.tsx

```diff
@@ -21,11 +21,11 @@ export const CtiDisabledModuleComponent = () => {
   const danger = useMemo(
     () => (
       <CtiInnerPanel
-        color={'danger'}
+        color={'warning'}
         title={i18n.DANGER_TITLE}
         body={i18n.DANGER_BODY}
         button={
-          <EuiButton href={threatIntelDocLink} color={'danger'} fill>
```

Comment:
`fill` was removed here which reduces the contrast between the button and the panel as can be seen in the screenshot. Should we add that back?

Reply:
I tried it with fill first but the bright yellow button with the black text looked kinda awkward IMHO - deferring this to @yiyangliu9286

Reply:
https://elastic.slack.com/archives/C7QC1JV6F/p1626083763403300?thread_ts=1625871367.401800&cid=C7QC1JV6F

Reply:
Thanks for confirming, it looks like EUI doesn't have clear differences/recommendation for using `fill` button for `EuiCallOut` or no, so I'm fine with removing the `fill` button @ecezalp.